models/demos/tg/falcon7b/demo_tg.py

```diff
@@ -10,9 +10,9 @@
 @pytest.mark.parametrize(
     "perf_mode, max_seq_len, expected_perf_metrics, greedy_sampling, expected_greedy_output_path",
     (
-        (True, 128, {"prefill_t/s": 5940, "decode_t/s": 1240, "decode_t/s/u": 1.21}, False, None),
-        (True, 1024, {"prefill_t/s": 12300, "decode_t/s": 1130, "decode_t/s/u": 1.10}, False, None),
-        (True, 2048, {"prefill_t/s": 9340, "decode_t/s": 1130, "decode_t/s/u": 1.10}, False, None),
+        (True, 128, {"prefill_t/s": 12760, "decode_t/s": 3510, "decode_t/s/u": 3.4}, False, None),
```

Comment:
2 questions: 
- What caused this more than 2x improvement?
- decode_t/s is 100x larger than decode_t/s/u, I am not getting this part.

Reply:
The perf improvement came from the switch to ttnn multi-device tensors. The t/s number is the t/s/u value x 1024 since that is the number of users running in parallel across the 32 chips.

Reply:
Nice, I love multi-device tensors! Props for introducing them to folks and to you for enabling it! Super convenient and much cleaner implementation!